common/src/main/java/com/wynntils/mc/event/DisplayResizeEvent.java

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.mc.event;
+
+import net.minecraftforge.eventbus.api.Event;
+
+public abstract class DisplayResizeEvent extends Event {
```

Comment:
I'm not really sure we need a Pre/Post version of this. I think in most cases we can get along with just a single event for all kinds of things that happens in vanilla. I think we should not create Pre/Post versions of everything "just in case", but only put in the separation if it turns out that we cannot do with a single event. For resizing this sounds unlikely to ever happen.